src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/SynthesizedSymbols/AnonymousType.TypeParameterSymbol.cs

```diff
@@ -93,7 +93,10 @@ public override bool HasValueTypeConstraint
 
             public override bool AllowByRefLike
             {
-                get { return false; }
+                get
+                {
+                    return _container.IsDelegateType() && _container.Manager.Compilation.SourceAssembly.RuntimeSupportsByRefLikeGenerics;
```

Comment:
>Manager.Compilation.SourceAssembly [](http://example.com/codeflow?start=69&length=34)

Consider simplifying to `ContainingAssembly`.

Reply:
> > Manager.Compilation.SourceAssembly
> 
> Consider simplifying to `ContainingAssembly`.

Will do for the next PR.